src-docs/src/views/text_utilities/wrapping.tsx

```diff
@@ -0,0 +1,206 @@
+          </EuiPanel>
+        }
+        snippet={`<div className="eui-textNoWrap">
+  /* Your content */
+</div>`}
+      />
+
+      <ThemeExample
+        title={<code>.eui-textTruncate</code>}
+        description={
+          <>
+            <p>
+              Truncates text at 100% width of its parent and will display an
+              ellipsis.
+            </p>
+            <p>
+              <strong>Tip:</strong> When truncating text, it is recommended to
+              include the full text within an HTML <EuiCode>title</EuiCode>{' '}
+              attribute or by wrapping the element within an{' '}
+              <strong>EuiToolTip</strong>.
+            </p>
+          </>
+        }
+        example={
+          <EuiPanel
+            color="warning"
+            style={{ maxWidth }}
+            className="eui-textTruncate"
+          >
+            This text will not to wrap but truncate beyond the boundaries of the
+            yellow box.
+          </EuiPanel>
+        }
+        snippet={`<div
+  className="eui-textTruncate"
+  title={Your content}>
+  /* Your content */
+</div>`}
+      />
+      {/* Mixin */}
+      {!showSass ? (
+        <ThemeExample
+          title={<code>euiTextTruncate()</code>}
+          description={
+            <>
+              <p>
+                Use this style function to apply{' '}
+                <EuiCode>.eui-textTruncate</EuiCode> within your CSS-in-JS
+                styling.
+              </p>
+              <p>
+                This utility accepts a single optional parameter for customizing
+                the width of the truncated text. If not passed, it defaults to{' '}
+                <EuiCode language="css">max-width: 100%;</EuiCode>.
+              </p>
+            </>
+          }
+          snippet={`\${euiTextTruncate()}
+  \${euiTextTruncate('150px')}`}
+          snippetLanguage="emotion"
+        />
+      ) : (
+        <ThemeExample
+          title={<code>Mixin</code>}
+          description={
+            <p>
+              Use this Sass mixin to apply <EuiCode>.eui-textTruncate</EuiCode>{' '}
+              within your classes. No parameters are taken for this utility.
+            </p>
+          }
+          snippet={'@include euiTextTruncate;'}
+          snippetLanguage="scss"
+        />
+      )}
+
+      <ThemeExample
+        title={<code>.eui-textBreakWord</code>}
+        description={
+          <p>
+            Wraps the text across lines like normal, but forces long words like
+            URLs to break.
+          </p>
+        }
+        example={
+          <EuiPanel
+            color="warning"
+            style={{ maxWidth }}
+            className="eui-textBreakWord"
+          >
+            This text will wrap like normal but this long link {longLink} will
+            break mid-word.
+          </EuiPanel>
+        }
+        snippet={`<div
+  className="eui-textTruncate"
+  title={Your content}>
+  /* Your content */
+</div>`}
```

Comment:
Looks like this is the wrong snippet
```suggestion
        snippet={`<div className="eui-textBreakWord">
  /* Your content */
</div>`}
```

Reply:
Great catch!! 🤦 https://github.com/elastic/eui/pull/5854/commits/ab3f6b163cb79fb406bf14f556e7ea9cf2ccdce7